Monos Chinos/dist/metadata.json

```diff
@@ -0,0 +1,18 @@
+{
+  "author": {
+    "name": "white",
+    "id": "289710422112731138"
+  },
+  "service": "Monos Chinos",
+  "description": {
+    "en": "Monos Chinos is a streaming website where you can watch anime with spanish subtitles."
+  },
+  "url": ["monoschinos.com"],
+  "version": "1.0.0",
```

Comment:
url shouldnt be in an array if its only 1

Reply:
corrected!